sootup.core/src/main/java/sootup/core/graph/MutableBlockStmtGraph.java

```diff
@@ -1625,4 +1625,106 @@ public List<Trap> buildTraps() {
     traps.sort(getTrapComparator(stmtsBlockIdx));
     return traps;
   }
+
+  @Override
+  public void removeTrap(@Nonnull Trap trap) {
```

Comment:
this seems a bit complicated - are you in the office tomorrow?

Reply:
Yes, we can discuss it tomorrow then.